app/fissile.go

```diff
@@ -452,8 +461,57 @@ func (f *Fissile) GeneratePackagesRoleImage(repository string, roleManifest *mod
 	return nil
 }
 
+// GeneratePackagesRoleTarball builds a tarball snapshot of the build context
+// for the docker image for the packages layer where all packages are included
+func (f *Fissile) GeneratePackagesRoleTarball(repository string, roleManifest *model.RoleManifest, noBuild, force bool, roles model.Roles, outputDirectory string, packagesImageBuilder *builder.PackagesImageBuilder) error {
+	if len(f.releases) == 0 {
+		return fmt.Errorf("Releases not loaded")
+	}
+
+	packagesLayerImageName, err := packagesImageBuilder.GetRolePackageImageName(roleManifest, roles)
+	if err != nil {
+		return fmt.Errorf("Error finding role's package name: %s", err.Error())
+	}
+	outputPath := filepath.Join(outputDirectory, fmt.Sprintf("%s.tar", packagesLayerImageName))
+
+	if !force {
+		file, err := os.Open(outputPath)
```

Comment:
You can't be using the file outside the `if !force` scope, so doesn't a stat make more sense than an open here?